flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/source/FileStoreSource.java

```diff
@@ -93,17 +93,22 @@ public Boundedness getBoundedness() {
             read.withDropDelete(false);
         }
 
+        int[][] valueCountProject = null;
```

Comment:
Nit: is there a better naming alternative for `valueCountProject`?

Reply:
valueCountModeProjects